elide-core/src/test/java/com/yahoo/elide/parsers/expression/PermissionToFilterExpressionVisitorTest.java

```diff
@@ -308,6 +356,47 @@ public boolean ok(Model object, RequestScope requestScope, Optional<ChangeSpec>
     static class GOOD8 {
     }
 
+    //Test NOT with "IN" FilterPredicate
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (owner is user)")
+    static class GOOD11 {
+    }
+
+    //Test NOT with "LT" FilterPredicate
+    @Entity
```

Comment:
These classes should be embedded inside their own test functions.  Each of these tests should be a separate TestNG function.